packages/typespec-azure-resource-manager/lib/backcompat.tsp

```diff
@@ -16,7 +16,7 @@ alias PrivateEndpointConnectionProperties = CommonTypes.PrivateEndpointConnectio
 alias PrivateLinkServiceConnectionState = CommonTypes.PrivateLinkServiceConnectionState;
 alias PrivateEndpointConnectionProvisioningState = CommonTypes.PrivateEndpointConnectionProvisioningState;
 alias PrivateEndpointServiceConnectionStatus = CommonTypes.PrivateEndpointServiceConnectionStatus;
-alias PrivateEndpointConnectionParameter = CommonTypes.PrivateEndpointConnectionParameter;
+alias PrivateEndpointConnectionName = CommonTypes.PrivateEndpointConnectionName;
```

Comment:
I am not sure about this breaking change fix. Can't we not apply `@encodedName()` to correct this only common-type generation script to get the matching swagger and have the -ref to point to the swagger name?

Reply:
I have tried this approach, and I am not getting the desired results, encodedName is not impacting at all the name, I will investigate further